content/integrate/redis-data-integration/installation/install-k8s.md

```diff
@@ -0,0 +1,472 @@
+
+The installation creates the following K8s objects:
+
+-   A K8s [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/) named `rdi`.
+-   [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) for the 
+    [RDI operator]({{< relref "/integrate/redis-data-integration/architecture#how-rdi-is-deployed" >}}),
+    [metrics exporter]({{< relref "/integrate/redis-data-integration/observability" >}}), and API server.
+-   A [service account](https://kubernetes.io/docs/concepts/security/service-accounts/) along with a
+    [role](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update)
+    and [role binding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding) for the RDI operator.
+-   A [Configmap](https://kubernetes.io/docs/concepts/configuration/configmap/)
+    for the different components with RDI Redis database details.
+-   [Secrets](https://kubernetes.io/docs/concepts/configuration/secret/)
+    with the RDI Redis database credentials and TLS certificates.
+
+You can use this installation on [OpenShift](https://docs.openshift.com/) and other K8s distributions
+including cloud providers' K8s managed clusters.
+
+You can pull the RDI images from
+[Docker Hub](https://hub.docker.com/) or from your own [private image registry](#using-a-private-image-registry).
+
+## Before you install
+
+Complete the following steps before running Helm:
+
+-   [Create the RDI database](#create-the-rdi-database) on your Redis Enterprise cluster.
+-   Create a [user]({{< relref "/operate/rs/security/access-control/create-users" >}})
+    for the RDI database if you prefer not to use the default password (see
+    [Access control]({{< relref "/operate/rs/security/access-control" >}}) for
+    more information).
+-   Download the RDI helm chart tar file from the [download center](https://cloud.redis.io/#rlec-downloads).
+-   If you want to use a private image registry,
+    [prepare it with the RDI images](#using-a-private-image-registry).
+
+### Create the RDI database
+
+RDI uses a database on your Redis Enterprise cluster to store its state
+information. *This requires Redis Enterprise v6.4 or greater*.
+
+- Use the Redis console to create a database with 250MB RAM with one primary and one replica.
+- If you are deploying RDI for a production environment then secure this database with a password
+  and [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security).
+
+You should then provide the details of this database in the [`values.yaml`](#the-valuesyaml-file)
+file as described below.
+
+### Using a private image registry
+
+Add the RDI images from [Docker Hub](https://hub.docker.com/) to your local registry.
+The example below shows how to specify the registry and image pull secret in the
+[`values.yaml`](#the-valuesyaml-file) file for the Helm chart:
+
+```yaml
```

Comment:
@spicy-sauce do I remember correctly that you told me that you had tried the second format and it didn't work?